WordPress/Classes/ViewRelated/Plugins/PluginDirectoryAccessoryView.swift

```diff
@@ -0,0 +1,180 @@
+import UIKit
+import Gridicons
+
+struct PluginDirectoryAccessoryView {
```

Comment:
I find it a bit odd that this is named `*View` but not a `UIView` subclass. If I read it right, all the methods that actually create the returned view, create a container first. Could `PluginDirectoryAccessoryView` be the container view and have every other method add content to it?

Reply:
I renamed it to `Item`, but I didn't want to make it a `UIView` subclass to make it impossible to instantiate it in a wrong way.